cvpysdk/backupset.py

```diff
@@ -663,6 +696,278 @@ def _update(self, backupset_name, backupset_description, default_backupset):
 
         return self._process_update_reponse(request_json)
 
+    @staticmethod
+    def _get_timestamp(formatted_time):
+        """ Returns the epoch timestamp given the input time is in format %Y-%m-%d %H:%M:%S
+        
+        Args:
+            formatted_time          (multiple)  --  String of formatted time or integer of timestamp
+            
+        Returns:
+            (int)- Timestamp

```

Comment:
No parenthesis should come around the return data type
Also, elaborate more on the return data

Follow same style as per the design